x-pack/legacy/plugins/lens/public/editor_frame_plugin/embeddable/embeddable_factory.ts

```diff
@@ -69,6 +68,12 @@ export class EmbeddableFactory extends AbstractEmbeddableFactory {
     input: Partial<EmbeddableInput> & { id: string },
     parent?: IContainer
   ) {
+    if (this.expressionRenderer === null) {
```

Comment:
This check can be done away with if we make the expression renderer part of the constructor.